front/lib/api/llm/clients/anthropic/utils/test/anthropic_to_events.test.ts

```diff
@@ -0,0 +1,151 @@
+import { describe, expect, it } from "vitest";
+
+import { streamLLMEvents } from "../anthropic_to_events";
+import { MessageStreamEvent } from "@anthropic-ai/sdk/resources/messages.mjs";
+import { ProviderMetadata } from "@app/lib/api/llm/types/events";
+
+async function* createAsyncGenerator<T>(items: T[]): AsyncGenerator<T> {
+  for (const item of items) {
+    yield item;
+  }
+}
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) ! We use it at 2 places now, so we can move it somewhere else; `front/tests/utils/utils.ts`